modules/lx-autoscale/watchmaker-lx-autoscale.template.cfn.yaml

```diff
@@ -556,6 +562,10 @@ Resources:
                 yum -y install unzip &&
                 unzip /etc/cfn/scripts/AmazonCloudWatchAgent.zip -d
                 /etc/cfn/scripts/aws-cw-agent
+            03-manipulate-cloudwatch-config:
+              command: >-
+                python3 Add-CWL-target.py
+              cwd: /etc/cfn/scripts/aws-cw-agent
```

Comment:
Any particular reason to run the script with cwd and a relative path, rather than using the full path in the command? i.e. `python3 /etc/cfn/scripts/aws-cw-agent/Add-CWL-target.py`?

Reply:
Saw it [here](https://github.com/plus3it/terraform-aws-watchmaker/pull/84/files#diff-89543a5acaa9dcdd3a588914c7d159c6L565) and just decided to follow suit. Updated to use a relative path.